data/i18n/en/en.toml

```diff
@@ -49,26 +49,17 @@ other = "CVE ID"
 [cve_issue_url]
 other = "CVE GitHub Issue URL"
 
-[cve_json_external_url]
-other = "external_url"
-
-[cve_json_id]
-other = "id"
-
-[cve_json_summary]
-other = "summary"
-
-[cve_json_url]
-other = "url"
-
 [cve_summary]
 other = "Issue Summary"
 
 [cve_table]
 other = "Official Kubernetes CVE List"
 
-[cve_url]
-other = "CVE URL"
+[cve_table_date_format]
```

Comment:
Identical with the main branch HEAD.

Ex:
https://github.com/kubernetes/website/blob/main/data/i18n/en/en.toml#L58